nmdc_server/config.py

```diff
@@ -41,6 +41,10 @@ class Settings(BaseSettings):
     orcid_client_secret: str = "oauth secret key"
     orcid_authorize_scope: str = "/authenticate"
 
+    # for zipstreamer
+    zip_streamer_url: str = "http://zipstreamer:4008/download"
+    zip_streamer_chunk_size_bytes: int = 512
```

Comment:
I think this should probably be somewhere between 1-5MB by default.

Reply:
Updated (fixed up into commit e280bed)